src/components/Gradebook/GradebookFilters/AssignmentTypeFilter/test.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { filterAssignmentType } from 'data/actions/grades';
+
+import {
+  AssignmentTypeFilter,
+  mapStateToProps,
+  mapDispatchToProps,
+} from '.';
+
+jest.mock('data/selectors/filters', () => ({
+  /** Mocking to use passed state for validation purposes */
+  selectableAssignmentLabels: jest.fn().mockImplementation((state) => ({
+    state,
+    selectableLabels: 'selectableLabels',
+  })),
+}));
+
+describe('AssignmentTypeFilter', () => {
+  let props = {
+    assignmentTypes: ['assignMentType1', 'AssigNmentType2'],
```

Comment:
are these random caps a thing you do on purpose? i notice them in other tests too.

Reply:
Yes.  Ensures we aren't getting algorithmig/auto text.  Just makes the tests a little bit more secure.